generic.json

```diff
@@ -35,5 +29,5 @@
         }
 
     },
-    "required": ["jsonrpc"]
+    "required": ["jsonrpc", "oneOf"]
```

Comment:
I am not sure you can do that! If I understand correctly, the `oneOf` construct (and any similar ones) will successively REPLACE the `oneOf` by one of the option inside of the construct. Thus the `"required": ["jsonrpc", "oneOf"]` will never be fulfilled